Gruntfile.js

```diff
@@ -23,22 +22,10 @@ module.exports = function(grunt) {
       production: 'publish',
       bamboo: 'bamboo',
 
-      // webrtc-adapter submodule
-      webrtcAdapterSubmodule: 'third_party/adapter/',
-      googleAdapterPath: '<%= webrtcAdapterSubmodule %>/out/adapter.js',
-
       pluginInfoRoot: grunt.option('pluginInfoRoot') || '<%= source %>',
       pluginInfoFile: grunt.option('pluginInfoFile') || 'pluginInfo.js',
 
-      githash: {
-        submodule: {
-          options: {
-            dir: '<%= webrtcAdapterSubmodule %>'
-          }
-        },
-      },
-
-      version: '<%= pkg.version %>-<%= githash.submodule.short %>',
+      version: '<%= pkg.version %>',
```

Comment:
I think that just having our version number is sufficient since we can document the webrtc/adapter version in the release notes for every release and its in the `package.json` file. 

But since this might change our release tagging process a bit, it might be better we reconfirm with @ncurrier on this.